src/select/SimpleSelect.js

```diff
@@ -727,7 +741,7 @@ class SimpleSelect extends Component {
           <div className="crane-select-outer-input">
             {beforeInput}
             <div
-              aria-controls={isOpen ? 'crane-select-menu-container' : null}
+              aria-owns={isOpen ? `crane-select-menu-${id}` : undefined}
```

Comment:
I went back and forth on this. Ideally it would be nice to have an internal store of dropdown ids so `id` isn't required... but for now I think you're right, `id` should be required. Fixed!